react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-alert */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import actionsCreators from '../../../redux/login/actions';
+
+import Login from './layout';
+
+class LoginForm extends Component {
+  handleSubmit = values => {
+    this.props.getLogin(JSON.stringify(values));
+  };
```

Comment:
You have to send only `values` to login and let the service take care of this 
on login service you're going to send the values like this -> `({ email, password })` instead of `values`

Reply:
yes, I already correct that on a new branch that I haven't pushed yet.